src/models/task/postgres.js

```diff
@@ -21,217 +21,228 @@ const TaskRepo = (postgres) => {
-        }
+  const getAllCompletedTasks = async (userID) => {
+    const values = [userID];
+    try {
+      const client = await postgres.connect();
+      const res = await client.query(getAllCompletedTasksSQL, values);
+      client.release();
+      return [res.rows, null]; // error message is empty string
+    } catch (err) {
+      return [null, 'Error at Completed Tasks']; // return null for the data if user's tasks DNE
     }
+  };
 
-    const getAllScheduledTasksSQL = `
+  const getAllScheduledTasksSQL = `
         SELECT * FROM tasks WHERE user_id=$1 AND scheduled=TRUE ORDER BY start_time;
     `;
 
-    const getAllScheduledTasks = async (userID) => {
-        const values = [userID];
-        try {
-            const client = await postgres.connect();
-            const res = await client.query(getAllScheduledTasksSQL, values);
-            client.release();
-            return [res.rows, null] // error message is empty string
-        } catch (err) {
-            return [null, "Error at Scheduled"]; // return null  for the data if user's tasks DNE
-        }
+  const getAllScheduledTasks = async (userID) => {
+    const values = [userID];
+    try {
+      const client = await postgres.connect();
+      const res = await client.query(getAllScheduledTasksSQL, values);
+      client.release();
+      return [res.rows, null]; // error message is empty string
+    } catch (err) {
+      return [null, 'Error at Scheduled']; // return null  for the data if user's tasks DNE
     }
+  };
 
-    const getAllNotScheduledTasksSQL = `
+  const getAllNotScheduledTasksSQL = `
         SELECT * FROM tasks WHERE user_id=$1 AND scheduled = FALSE ORDER BY updated_time DESC;
     `;
 
-    const getAllNotScheduledTasks = async (userID) => {
-        const values = [userID];
-        try {
-            const client = await postgres.connect();
-            const res = await client.query(getAllNotScheduledTasksSQL, values);
-            client.release();
-            return [res.rows, null] // error message is empty string 
-        } catch (err) {
-            return [null, "Error at Not Scheduled"]; // return null  for the data if user's tasks DNE
-        }
+  const getAllNotScheduledTasks = async (userID) => {
+    const values = [userID];
+    try {
+      const client = await postgres.connect();
+      const res = await client.query(getAllNotScheduledTasksSQL, values);
+      client.release();
+      return [res.rows, null]; // error message is empty string
+    } catch (err) {
+      return [null, 'Error at Not Scheduled']; // return null  for the data if user's tasks DNE
     }
+  };
 
-    const getTasksForSchedulingSQL = `
-        SELECT * FROM tasks WHERE user_id=$1 AND id = ANY($2) AND scheduled = FALSE ORDER BY due_date ASC;
+  const getTasksForSchedulingSQL = `
+        SELECT * FROM tasks WHERE user_id=$1 AND id = ANY($2) AND scheduled = FALSE ORDER BY due_date, duration;
```

Comment:
Order by duration so that shorter tasks are consistently returned first for scheduling.